clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java

```diff
@@ -519,6 +519,10 @@ String insertHeader(Context context, JSONArray arr,@Nullable final String caller
             }
             // Add frs (First Request in Session) and update first request flag
             header.put("frs", coreMetaData.isFirstRequestInSession());
+
+            if(config.getDebugLevel() == 3)
```

Comment:
@Anush-Shand Just add comment, mentioning use of this header.

Reply:
@Anush-Shand is there a possibility of config.getDebugLevel being NULL?

Reply:
@darshanclevertap Nope, I don't think so